### PR TITLE
created EventCard and also refactored component folder

### PR DESCRIPTION
**Pull request by SlimPortfolio**

### CODE REVIEW

File: client/src/Components/Login.jsx [hunk 1]
```diff
@@ -53,7 +53,6 @@ function Login() {
                             <InputLabel htmlFor="username">Username</InputLabel>
                             <OutlinedInput
                                 id="username"
-                                type={showPassword ? 'text' : 'password'}
                                 label="Username"
                             />
```
Comment (StreetsJ) [inline client/src/Components/Login.jsx:57]: Why did you remove the type?

Comment (SlimPortfolio) [reply client/src/Components/Login.jsx:57]: this was under the username section. So it was giving the option to show/hide the username
